### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ardeeshany**
Merge Dotto Page

### CODE REVIEW

File: post.html [hunk 1]
```diff
@@ -840,6 +834,7 @@ <h5 class="card-title mb-0">Amin Zamani</h5>
         </div>
       </section>
       <section
+        v-if="activeDotto === '1'"
```
Comment (ardeeshany) [inline post.html:837]: Hi @ItsAminZamani  We need to update the URL here by activating each Dotto. We can use the `Dotto` id here that I added into the most recent JSON file.

**Review by ardeeshany — COMMENTED** [review]
As we discussed, we may need to get rid of `resizable` in this version (if there is no final robust solution for that) and replace it with the maximizable feature.

**Comment by ardeeshany** [comment]
@ItsAminZamani Is it better to save the JSON into a file and read it from the HTML file? We need a robust solution that works in offline cases.

**Comment by ItsAminZamani** [comment]
Yes, it is more clean way.

**Pull request merged**